lute/themes/css/Apple_Books.css

```diff
@@ -1,10 +1,9 @@
 body {
-  background-color: #f8f1e2; /* beige paper */
+  --background-color: #f8f1e2;
+  background-color: var(--background-color); /* beige paper */
   color: #262523; /* dark grey font */
 }
 
- #reading_menu { background-color: #f8f1e2; }
```

Comment:
Was this style redundant?  Wonder why it was there if yes.  Same comment applies to the other themes.

Also, did the themes need to change to support this PR, or is this additional/related work that you're cleaning up?  Minor point, just asking.

Reply:
I applied the color in the `styles.css` via the variable, so this became reduntant. by default it's just white, but changing the `--background-color` in these theme files changes the reading_menu color